templates/ts-consumer/package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "@scramjet/ts-simple",
+  "name": "@scramjet/ts-consumer",
   "version": "1.0.0",
   "description": "Scramjet Cloud Platform Sequence Typescript Template",
   "main": "index.js",
   "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
+    "predeploy" : "npm i && tsc -p tsconfig.json && cp -r package.json dist/ && mv node_modules dist"
```

Comment:
this command will install all dependencies along with devDependencies, and move them to sequence package. Only dependencies should be added to sequence package. Proper command would be `tsc -p tsconfig.json && cp package.json dist/ && (cd dist && npm i --omit=dev)`